README.md

```diff
@@ -197,7 +197,73 @@ data-validation (--verbose or -v) (--log-level or -ll) validate row
+  --tables-list or -tbls SOURCE_SCHEMA.SOURCE_TABLE=TARGET_SCHEMA.TARGET_TABLE
+                        Comma separated list of tables in the form schema.table=target_schema.target_table
+                        Target schema name and table name are optional.
+                        i.e 'bigquery-public-data.new_york_citibike.citibike_trips'
+  --primary-keys PRIMARY_KEYS, -pk PRIMARY_KEYS
+                        Comma separated list of primary key columns 'col_a,col_b'
+  --comparison-fields or -comp-fields FIELDS
+                        Comma separated list of columns to compare. Can either be a physical column or an alias
+                        See: *Calculated Fields* section for details
+  --hash COLUMNS        Comma separated list of columns to hash or * for all columns 
+  --concat COLUMNS      Comma separated list of columns to concatenate or * for all columns (use if a common hash function is not available between databases)
+  --config-dir CONFIG_DIR, -cdir CONFIG_DIR
+                        Directory Path to store YAML Config Files
+  --partition-num [1-1000], -pn [1-1000]
+                        Number of partitions/config files to generate
+                        In case this value exceeds the row count of the source/target table, its will be decreased to max(source_row_count, target_row_count)
+  [--partition-key PARTITION_KEY, -partkey PARTITION_KEY]
+                        Column on which the partitions would be generated. Column type must be integer. Defaults to Primary key
+  [--threshold THRESHOLD, -th THRESHOLD]
+                        Float max threshold for percent difference
+  [--filters SOURCE_FILTER:TARGET_FILTER]
+                        Colon spearated string values of source and target filters.
+                        If target filter is not provided, the source filter will run on source and target tables.
+                        See: *Filters* section
+  [--use-random-row, -rr]
+                        Finds a set of random rows of the first primary key supplied.
+  [--random-row-batch-size RANDOM_ROW_BATCH_SIZE, -rbs RANDOM_ROW_BATCH_SIZE]
+                        Row batch size used for random row filters (default 10,000).
+  [--result-handler-config RESULT_HANDLER_CONFIG, -rc RESULT_HANDLER_CONFIG]
+                        Result handler config details
+  [--bq-result-handler or -bqrh PROJECT_ID.DATASET.TABLE]
+                        BigQuery destination for validation results. Defaults to stdout.
+                        See: *Validation Reports* section
+  [--labels LABELS, -l LABELS]
+                        Key value pair labels for validation run
+  [--service-account SERVICE_ACCOUNT, -sa SERVICE_ACCOUNT]
+                        Path to SA key file for result handler output
+  [--format FORMAT, -fmt FORMAT]
+                        Set the format for printing command output, Supported formats are (text, csv, json, table). Defaults to table
+  [--filter-status FILTER_STATUS, -fs FILTER_STATUS]
```

Comment:
We don't need any of the above flags (line 252-265) as we aren't running validations, just creating configs.